tests/sqllogictests/suites/udf_server/udf_server_test

```diff
@@ -0,0 +1,363 @@
+# Please start the UDF Server first before running this test:
+#   python3 tests/udf-server/udf_test.py
+#
+
+statement ok
+set enable_udf_server = 1;
```

Comment:
is this config considered as a dyanmic settings which can be enabled by any user? 

IMHO it'd be better to limit this configuration in the config file, and can not be changed in the runtime.

Reply:
done